howto-gallery/behavior_packs/howto-gallery/manifest.json

```diff
@@ -4,23 +4,35 @@
     "name": "Minecraft JavaScript How-to Gallery",
     "description": "A gallery and GameTest of many code snippets",
     "uuid": "7de074b8-9052-4b1c-9342-b5c4280269fa",
-    "version": [0, 0, 1],
-    "min_engine_version": [1, 20, 0]
+    "version": [
+      0,
+      0,
+      1
+    ],
+    "min_engine_version": [
+      1,
+      20,
+      0
+    ]
   },
   "modules": [
     {
       "description": "Script resources",
       "type": "script",
       "language": "javascript",
       "uuid": "df6523d1-4442-45c2-b33a-1c882d388683",
-      "version": [0, 0, 1],
+      "version": [
+        0,
+        0,
+        1
+      ],
       "entry": "scripts/main.js"
     }
   ],
   "dependencies": [
     {
       "module_name": "@minecraft/server",
-      "version": "1.7.0-beta"
+      "version": "1.9.0-beta"
```

Comment:
Probably here and in other cases you reference beta, this should point at the current latest beta. Alternatively, we should  target stable in the samples.

Reply:
Fixed, but howto-gallery depends on beta features. Leaving beta in this project.